helmfile.d/upstream/falcosecurity/falco/values.yaml

```diff
@@ -444,6 +543,48 @@ falcoctl:
         # -- See the fields of the artifact.install section.
         pluginsDir: /plugins
 
+# -- serviceMonitor holds the configuration for the ServiceMonitor CRD.
+# A ServiceMonitor is a custom resource definition (CRD) used to configure how Prometheus should
+# discover and scrape metrics from the Falco service.
+serviceMonitor:
+  # -- create specifies whether a ServiceMonitor CRD should be created for a prometheus operator.
+  # https://github.com/coreos/prometheus-operator
+  # Enable it only if the ServiceMonitor CRD is installed in your cluster.
+  create: false
```

Comment:
Did you look at enabling this and what metrics it produces?

Reply:
I looked at the docs for it but didn't do a deeper investigation, I considered that out of scope for this task. I believe @viktor-f made note of it as well.